jre8/plan.sh

```diff
@@ -76,7 +76,7 @@ do_install() {
   build_line "Setting rpath for '${pkg_prefix}/bin/java' to '$LD_RUN_PATH'"
 
   LD_RUN_PATH=$LD_RUN_PATH:$pkg_prefix/lib/amd64/jli:$pkg_prefix/lib/amd64/server:$pkg_prefix/lib/amd64
-  LD_RUN_PATH=$LD_RUN_PATH:$pkg_prefix/jre/lib/amd64/jli:$pkg_prefix/jre/lib/amd64/server:$pkg_prefix/jre/lib/amd64
+  #LD_RUN_PATH=$LD_RUN_PATH:$pkg_prefix/jre/lib/amd64/jli:$pkg_prefix/jre/lib/amd64/server:$pkg_prefix/jre/lib/amd64
```

Comment:
extra comment?

Reply:
Yeah,  forgot to remove those after validating the updated paths  were correct and executables had all linked correctly. Removed!